src/Apps/Artwork/Components/OtherWorks/index.tsx

```diff
@@ -152,7 +139,7 @@ export const OtherWorksFragmentContainer = createFragmentContainer(
   {
     artwork: graphql`
       fragment OtherWorks_artwork on Artwork {
-        contextGrids {
+        contextGrids(includeRelatedArtworks: false) {
```

Comment:
Extracting the `RelatedWorks` rail from the frontend in Force into its own QueryRenderer wasn't sufficient to move the expensive related artworks query out of `OtherWorks`. 

We needed to additionally deal with the opinionated backend logic in MP that _also_ fetched those related works in order to plop them into this `contextGrids` response.

Now we can opt out of that expensive step, thanks to https://github.com/artsy/metaphysics/pull/4545